### PR TITLE
fix(extension): fix asset item sizings

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] [JIRA - \<link>](https://input-output.atlassian.net/browse/LW-6138)
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

apply proper stylings (**_width would be a bit different in case the scrollbar is there*_**)

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Screenshot 2023-06-13 at 18 05 45

Screenshot 2023-06-13 at 18 08 18

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/383/5259927156/index.html) for [56c7e970](https://github.com/input-output-hk/lace/pull/120/commits/56c7e970f7cd767aaa83501af59a869bf18df362)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 14     | 5      | 0       | 0     | 19    | ❌     |

### CODE REVIEW

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
 export type InfiniteScrollableTableProps = Omit<React.ComponentProps<typeof Table>, 'children'> & {
   infitineScrollProps: Omit<InfiniteScrollProps, 'children' | 'loader'>;
+  infinitScrollContainerClass?: string;
   loaderConfig?: SkeletonProps;
 };
 
 const TableBodyWrapper = ({
   children,
   loaderConfig,
+  infinitScrollContainerClass,
```
Comment (mchappell) [inline packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx:17]:
```suggestion
  infiniteScrollContainerClass,
```

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
   ...props
-}: Omit<InfiniteScrollProps, 'loader'> & { loaderConfig?: SkeletonProps }) => (
-  <InfiniteScroll className={styles.infinitScrollContainer} {...props} loader={<Skeleton {...loaderConfig} />}>
+}: Omit<InfiniteScrollProps, 'loader'> & { loaderConfig?: SkeletonProps; infinitScrollContainerClass?: string }) => (
```
Comment (mchappell) [inline packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx:19]:
```suggestion
}: Omit<InfiniteScrollProps, 'loader'> & { loaderConfig?: SkeletonProps; infiniteScrollContainerClass?: string }) => (
```

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
+  <InfiniteScroll
+    className={cn(styles.infinitScrollContainer, { [infinitScrollContainerClass]: infinitScrollContainerClass })}
```
Comment (mchappell) [inline packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx:21]:
```suggestion
    className={cn(styles.infiniteScrollContainer, { [infiniteScrollContainerClass]: infiniteScrollContainerClass })}
```

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
+    {...props}
+    loader={<Skeleton {...loaderConfig} />}
+  >
     {children}
   </InfiniteScroll>
 );
 
 export const InfiniteScrollableTable = ({
   infitineScrollProps,
+  infinitScrollContainerClass,
```
Comment (mchappell) [inline packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx:31]:
```suggestion
  infiniteScrollContainerClass,
```

**Review by mchappell — CHANGES_REQUESTED** [review]
A few typos to fix and looks good to hand off

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.module.scss [hunk 5]
```diff
@@ -79,6 +79,6 @@
 }
 .infinitScrollContainer {
```
[the remark this answers is not in the document]
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.module.scss]: resolved

File: packages/core/src/ui/components/AssetTable/AssetTable.tsx [hunk 6]
```diff
@@ -107,13 +107,18 @@ export const AssetTable = ({
       key: 'price'
     });
 
+  const [hasScollBar, setHasScrollBar] = useState<boolean>(false);
```
[the remark this answers is not in the document]
Comment (vetalcore) [reply packages/core/src/ui/components/AssetTable/AssetTable.tsx]: resolved

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 7]
```diff
@@ -1,30 +1,42 @@
...
   loaderConfig,
   ...props
 }: InfiniteScrollableTableProps): React.ReactElement => (
-  <TableBodyWrapper {...infitineScrollProps} loaderConfig={loaderConfig}>
+  <TableBodyWrapper
+    {...infitineScrollProps}
+    loaderConfig={loaderConfig}
+    infinitScrollContainerClass={infinitScrollContainerClass}
```
[the remark this answers is not in the document]
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx]: resolved

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx (hunk 4 above)
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx]: resolved

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx [hunk 8]
```diff
@@ -1,30 +1,42 @@
...
 
 export const InfiniteScrollableTable = ({
   infitineScrollProps,
+  infinitScrollContainerClass,
   loaderConfig,
   ...props
 }: InfiniteScrollableTableProps): React.ReactElement => (
-  <TableBodyWrapper {...infitineScrollProps} loaderConfig={loaderConfig}>
+  <TableBodyWrapper
+    {...infitineScrollProps}
```
[the remark this answers is not in the document]
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx]: resolved

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx (hunk 2 above)
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx]: resolved

File: packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx (hunk 1 above)
Comment (vetalcore) [reply packages/common/src/ui/components/Table/InfiniteScrollableTable.tsx]: resolved

**Review by mchappell — APPROVED** [review]

**Review by wklos-iohk — APPROVED** [review]

**Pull request merged**